src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.js

```diff
@@ -33,6 +33,7 @@ function resetFreePlanBankAccount(bankAccountID, session) {
                         shouldShowResetModal: false,
                         isLoading: true,
                         pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+                        achData: null,
```

Comment:
```suggestion
                        achData: {},
```

or

```suggestion
                        achData: ReimbursementAccountProps.reimbursementAccountDefaultProps.achData,
```

Reply:
I don't think either of those work. We're calling Onyx.merge here and that'd just merge an empty object with the existing one, which results in the existing object. See video below with your first suggestion:

https://github.com/Expensify/App/assets/22219519/9862f021-c69e-4351-9abd-2dd97ccb62ef


 We actually want to reset the whole object which we do by merging `null`.

Reply:
I was concerned about `achData: null` just because I am not sure it will crash on some places where `lodashGet(props.reimbursementAccount, 'achData') || {}` code is used because null doesn't fall back to {}

Reply:
I only see that in [EnableStep](https://github.com/Expensify/App/blob/648461bf011af7aeabc877034e621bc6c223fab4/src/pages/ReimbursementAccount/EnableStep.js#L51), which is the page in the OP's video. Doesn't seem to crash.

Reply:
Sorry, I updated comment. I meant `, {}`, not ` || {}`

Reply:
I think that should be taken care of by this - https://github.com/Expensify/App/blob/648461bf011af7aeabc877034e621bc6c223fab4/src/pages/ReimbursementAccount/ReimbursementAccountPage.js#L108-L111